### ISSUE TITLE
Feature request: apply a test to group of models

### ISSUE THREAD

**Issue by remilepriol** [z3z1ma/dbt-osmosis#94]
Hello, 

I would like to apply some tests to a subset of my models easily. I think the programmatic handling of yaml files by dbt osmosis could enable this, for instance by specifying some tests directly in the `dbt_project.yml` and having them applied when running

    dbt-osmosis yaml refactor

Does this idea interest other people ? could it fit in dbt-osmosis ?

**Comment by syou6162** [comment]
Technically, it would not be difficult to propagate the information from the test, but it does not appear to be currently being incorporated as intended.

- https://github.com/z3z1ma/dbt-osmosis/blob/main/src/dbt_osmosis/core/osmosis.py#L813-L814C25

This is my personal opinion, but I think it is better not to incorporate it by default. It would be nice if the `--add-tests` option could be added.

**Comment by z3z1ma** [comment]
Yeah it seems a very specific situation I would want to propagate tests everywhere. Firstly the properties of columns like nullability and uniqueness may change across transformations for example during denormalization.

This would need to be opt in and I am not sure the best way other than adding a flag and leaving it to use discretion to use responsibly. Perhaps a whitelist of `column.test` keys which are propagatable.

**Comment by z3z1ma** [comment]
Closing this unless people really care about it. I dont see test inheritance making a lot of sense. Anyone can re-open or make a case / PR for it though.
